subsys/testsuite/ztest/include/zephyr/ztest.h

```diff
@@ -22,19 +22,6 @@
 #error "You need to add CONFIG_ZTEST to your config file."
 #endif
 
-#ifndef KERNEL
-#define ARCH_STACK_PTR_ALIGN 8
-/* FIXME: Properly integrate with Zephyr's arch specific code */
-#ifdef __cplusplus
-extern "C" {
-#endif
-struct esf;
-typedef struct esf z_arch_esf_t;
-#ifdef __cplusplus
-}
-#endif
-#endif /* KERNEL */
-
```

Comment:
removed as they don't seem required by upstream CI, can be split into another commit